gradle/dependencies-graph.gradle

```diff
@@ -0,0 +1,26 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath pluginDependencies.dependencyGraph
+    }
+}
+
+import com.vanniktech.dependency.graph.generator.DependencyGraphGeneratorPlugin
+import com.vanniktech.dependency.graph.generator.DependencyGraphGeneratorExtension.Generator
+
+
+plugins.apply(DependencyGraphGeneratorPlugin)
+
+def mapboxGenerator = new Generator(
+        "mapboxLibraries", // Suffix for our Gradle task.
+        "", // Root suffix that we don't want in this case.
```

Comment:
As @vanniktech suggested here https://github.com/vanniktech/gradle-dependency-graph-generator-plugin/issues/41#issuecomment-382651675 it's not necessary to include the root suffix if we're not going to use it.

Reply:
Yes and it's also no longer present in 0.4.0 which means you have to remove it anyways.

Reply:
Awesome @vanniktech thanks for confirming!